### ISSUE TITLE
Error while evaluating internal scoped discriminated union

### ISSUE THREAD

**Issue by VTJDailey** [SwensenSoftware/unquote#123]
```
            module internal submodule =
                type TTEST = 
                | TTEST1
                | TTEST2

            [<Fact>]
            let ``performtest`` () =
                test <@ submodule.TTEST1 = submodule.TTEST1 @>

```
This throws a KeyNotFoundException.  Removing the internal modifier to submodule allows the test to run as expected
: 
Result Message:	
TTEST1 = TTEST1
System.Collections.Generic.KeyNotFoundException: The given key was not present in the dictionary.
   at Microsoft.FSharp.Collections.MapTreeModule.find[TValue,a](IComparer`1 comparer, TValue k, MapTree`2 m)
   at Microsoft.FSharp.Reflection.Impl.getUnionTagConverter@455-2.Invoke(Int32 tag)
   at Microsoft.FSharp.Reflection.Impl.getUnionCaseConstructorMethod(Type typ, Int32 tag, BindingFlags bindingFlags)
   at Microsoft.FSharp.Reflection.Impl.getUnionCaseConstructor(Type typ, Int32 tag, BindingFlags bindingFlags)
   at Microsoft.FSharp.Reflection.FSharpValue.MakeUnion(UnionCaseInfo unionCase, Object[] args, FSharpOption`1 bindingFlags)
   at Microsoft.FSharp.Collections.IEnumerator.map@111.DoMoveNext(b& )
   at Microsoft.FSharp.Collections.IEnumerator.MapEnumerator`1.System-Collections-IEnumerator-MoveNext()
   at System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   at Microsoft.FSharp.Collections.SeqModule.ToArray[T](IEnumerable`1 source)
   at Swensen.Unquote.Evaluation.eval@146(FSharpList`1 env, FSharpExpr expr)
   at Swensen.Unquote.Evaluation.eval(FSharpList`1 env, FSharpExpr expr)
   at Swensen.Unquote.Evaluation.eval(FSharpList`1 env, FSharpExpr expr)
   at Swensen.Unquote.Reduction.reduce(FSharpList`1 env, FSharpExpr expr)
   at Swensen.Unquote.Reduction.loop@126(FSharpList`1 env, FSharpExpr expr, FSharpList`1 acc)

Expected: True
Actual:   False

**Comment by stephen-swensen** [comment]
Hi @VTJDailey - this issue was actually fixed recently by #122. It hasn't been released yet, but you can preview the CI build: https://ci.appveyor.com/project/stephen-swensen/unquote/build/artifacts

Regards,

Stephen

**Comment by VTJDailey** [comment]
Oh, wonderful, thanks!
